routers/web/repo/pull.go

```diff
@@ -652,6 +652,24 @@ func PrepareViewPullInfo(ctx *context.Context, issue *issues_model.Issue) *git.C
 	}
 
 	if pb != nil && pb.EnableStatusCheck {
+
+		var missingRequiredChecks []string
+		for _, requiredContext := range pb.StatusCheckContexts {
+			contextFound := false
+			for _, presentStatus := range commitStatuses {
+				if presentStatus.Context == requiredContext {
```

Comment:
`requiredContext` maybe a global express, I think it should also be handled here?